doc/project/release_process.rst

```diff
@@ -72,14 +72,17 @@ code base has begun.
 Stabilization Phase
 *******************
 
-Over the next weeks, only patches which fix problems should be submitted to the
-mainline.  On occasion, a more significant change will be allowed, but such
-occasions are rare and require a TSC approval (Change Control Board). As a
-general rule, if you miss submitting your code during the development phase for
-a given feature, the best thing to do is to wait for the next development cycle.
-(An occasional exception is made for drivers for previously unsupported
-hardware; if they do not touch any other in-tree code, they cannot cause
-regressions and should be safe to add at any time).
+Over the next weeks and depending on the release milestone, only stabilization,
+cosmetic changes, tests, bug and doc fixes are allowed (See :ref:`table
+<release_milestones>` below).
+
+On occasion, more significant changes and new features will be allowed, but such
+occasions are rare and require a TSC approval and a justification. As a general
+rule, if you miss submitting your code during the development phase for a given
+feature, the best thing to do is to wait for the next development cycle. (An
+occasional exception is made for drivers for previously unsupported hardware; if
+they do not touch any other in-tree code, they cannot cause regressions and
+should be safe to add at any time).
```

Comment:
this points sounds like any new driver could get in during stabilization as long as the PR is only for it, is that the case? Do we want it?

Reply:
still an exception that needs to be approved. This is an example. Very old text, probably needs rewording.

Reply:
how about dropping it? seems like an incentive for people to ask for exceptions

Reply:
Since focus really should be on testing and fixing, we should actively discourage developing new drivers/features during this phase :) dropping any mention of exceptions should help discourage asking for one. If something really is that important, I'm sure a request will find its way to the TSC regardless of what the doc says :D

Reply:
I would rather see the text:
> (An occasional exception is made for drivers for previously unsupported hardware; if
they do not touch any other in-tree code, they cannot cause regressions and
should be safe to add at any time).

replaced with something about how to properly prepare an exception request and make it clear that there has to really be an outstanding reason for the exception.

Reply:
I've always just viewed the "exceptions" for things that had been submitted well before the RC1 date but for reasons was taking a bit longer for the review cycles to complete. Not suggesting we try to word things this way though because it make our documentation denser then it already is. But I agree with @fabiobaltieri that we should just drop the parenthesis section from the paragraph.

Reply:
Yeah, could drop it.

I think it would be nice to include the steps for requesting an exception, even if it's a few bullet points.

Reply:
I am fine removing this, but in this PR I am not adding that and the PR is not about exceptions, so this can come in followup PR, be my guest.